### DISCUSSION TITLE
How to configure eslint for a mono repo ?

### DISCUSSION THREAD

**Discussion by SummySumanth** [eslint/eslint#18958] [Q&A]
I have a mono repo, having both frontend and backend in same repository, I want to specify eslint config in respective folders and have those apply in their directory itself (backend config in backend and frontend config in frontend), but eslint plugin in vscode keeps picking `.eslintrc.json` file instead of `eslint.config.js`, how to fix this ?

`my project structure is as following`

```
/.vscode
  └── settings.json
/app
  └── .vite
/backend
  └── eslint.config.js
  └── (files)
/frontend
  └── eslint.config.js
  └── (files)
  ```
  
 `./vscode/settings.json`
  ``` json
  {
    "eslint.workingDirectories": [
        {
            "directory": "../frontend",
            "changeProcessCWD": true,
            "overrideConfigFile": "./../frontend/eslint.config.js"
        },
        {
            "directory": "../backend",
            "changeProcessCWD": true,
            "overrideConfigFile": "./../backend/eslint.config.js"
        }
    ],
}
  ```
  
  `Error in vscode plugin output`
  ```
  [Error - 02:13:46] Request textDocument/diagnostic failed.
  Message: Request textDocument/diagnostic failed with message: Cannot read config file: /project/backend/.eslintrc.json
Error: ENOENT: no such file or directory, open '/project/backend/.eslintrc.json'
  Code: -32603 
  
  ```

**Comment by fasttime** [comment]
Hi @SummySumanth! This appears to be an issue with the VSCode ESLint extension, which we don't maintain. If this is the case, I would recommend you to report the problem at https://github.com/microsoft/vscode-eslint/issues.
